Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM debian:testing
+
+MAINTAINER Raniere Silva
+
+RUN apt-get -y update && apt-get -y install curl r-base r-base-dev
+RUN curl https://repo.continuum.io/archive/Anaconda3-4.2.0-Linux-x86_64.sh -o Anaconda3-4.2.0-Linux-x86_64.sh && bash Anaconda3-4.2.0-Linux-x86_64.sh -b
```

Comment:
No need for the temporary file, you can just pipe straight into a shell.  Although this is putting a lot of trust in HTTPS and the remote host; a safer approach is to put something like [hashpipe][1] between the network and the shell in the pipe.

[1]: http://jbenet.github.io/hashpipe/